LBH.AdultSocialCare.Api/V1/Boundary/ResidentialCare/Request/CarePackageForCreationRequest.cs

```diff
@@ -0,0 +1,32 @@
+using LBH.AdultSocialCare.Api.V1.Validations;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LBH.AdultSocialCare.Api.V1.Boundary.ResidentialCare.Request
```

Comment:
How do you get `packageType` via body, query, etc?

Reply:
Oh no. I originally intended to have different endpoints and identify from that, but looks like I have to add packageType